country-a-service/cda-generator/src/main/java/dk/sundhedsdatastyrelsen/ncpeh/cda/Oid.java

```diff
@@ -88,7 +88,7 @@ public enum Oid {
     ATC("2.16.840.1.113883.6.73", "Anatomical Therapeutic Chemical"),
 
     /// eHDSIHealthcareProfessionalRole used for function code
-    FUNCTION_CODE("1.3.6.1.4.1.12559.11.10.1.3.1.42.1", "eHDSIHealthcareProfessionalRole"),
+    FUNCTION_CODE("2.16.840.1.113883.2.9.6.2.7", "ISCO"),
```

Comment:
(this can be addressed in a follow-up)

```suggestion
    /// International standard classification of occupations (ISCO) used for function code
    ISCO("2.16.840.1.113883.2.9.6.2.7", "ISCO"),
```